jobs/ml_jobs/item_linkage/requirements.txt

```diff
@@ -0,0 +1,18 @@
+google-cloud-secret-manager==2.20.0
+google-cloud-bigquery[pandas]
+gcsfs>= 2023.1.0
+numpy==1.26.0
+pandas==2.2.2
+mlflow==2.14.1
+typer==0.7.0
+tqdm==4.64.1
+protobuf>=3.9.2
+loguru==0.6.0
+polars==0.18.6
+docarray==0.21.0
+lancedb==0.8.2
+hnne==0.1.10
+sentence_transformers==3.0.1
+streamlit==1.35.0
+recordlinkage==0.16
```

Comment:
requirements.in et requirements.txt

https://www.notion.so/passcultureapp/Comment-versionner-mes-packages-python-dans-un-microservice-c47a8f5364cf4201b5ed107228188ca9